tests/mod.rs

```diff
@@ -0,0 +1,6 @@
+mod block_range_scanner;
+mod common;
+mod historic;
+mod latest_events;
+mod live;
+mod sync;
```

Comment:
```suggestion
mod common;
mod historic;
mod latest_events;
mod live;
mod sync;
```
because block_range_scanner is at the root of `tests/` dir, it is picked up automatically by the test runner, no need to declare it in tests/mod.rs